packages/wallets/src/components/WalletsErrorScreen/WalletsErrorScreen.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import IcCashierError from '../../public/images/ic-cashier-error.svg';
+import WalletsActionScreen from '../WalletsActionScreen/WalletsActionScreen';
+
+type TProps = {
+    message: string;
+};
+
+const ErrorState: React.FC<TProps> = ({
```

Comment:
We have walleterror component already. cant we use that instead ?

Reply:
I will refactor the component a bit, but we dont need this component later i guess